docs/storage/storage-solution.md

```diff
@@ -2,25 +2,24 @@
 
 ## Concept
 
-Ability to run stateful workloads is a must. At the same time running stateful workloads is complex. Here the complexity is on another level considering the multi-/hybrid-cloud environment. Therefore, Claudie needs to be able to accommodate the stateful workloads, regardless of the underlying infrastructure providers.
+The ability to run stateful workloads is a must. At the same time, running stateful workloads is a complex task, even more so when considering the {multi,hybrid}-cloud environment. Therefore, Claudie needs to be able to accommodate the stateful workloads, regardless of the underlying infrastructure providers.
```

Comment:
Honestly, I'd just remove this line. I feel like this sort of rubbish doesn't belong here. Thoughts ?

Reply:
this has in fact been updated quite recently. We could perhaps make it 2 sentences instead of 2..